backend/src/Edna/Library/Web/Schema.hs

```diff
@@ -0,0 +1,11 @@
+module Edna.Library.Web.Schema
+  () where
```

Comment:
Btw, if export list is empty, our style guide permits writing `()` on the same line as `module …` and I think it looks a bit neater. But that's up to you (and I propose to remove this module anyway).